tensorflow/lite/kernels/bidirectional_sequence_lstm.cc

```diff
@@ -601,7 +597,7 @@ TfLiteStatus Prepare(TfLiteContext* context, TfLiteNode* node) {
   }
   TF_LITE_ENSURE_OK(context, context->ResizeTensor(context, bw_scratch_buffer,
                                                    bw_scratch_buffer_size));
-  if (is_hybrid_op) {
+  if (IsHybridOp(input, fw_input_to_output_weights)) {
```

Comment:
I think we just need call IsHybridOp once and store the result in is_hybrid_op, like the previous code do.